translator/translate/otel/processor/metricstransformprocessor/translator.go

```diff
@@ -89,6 +103,17 @@ func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 			},
 		}
 
+		if awscontainerinsight.EnhancedContainerInsightsEnabled(conf) {
+			for oldNvmeMetric, newNvmeMetric := range renameMapForNvme {
+				transformRules = append(transformRules, map[string]interface{}{
+					"include":  oldNvmeMetric,
+					"action":   "update",
+					"new_name": newNvmeMetric,
+				})
```

Comment:
```suggestion
				transformRules = append(transformRules, map[string]interface{}{
					"include":  oldNvmeMetric,
					"action":   "update",
					"new_name": newNvmeMetric,
					"operations": []map[string]interface{}{{
						"action": "add_label",
						"new_label": containerinsightscommon.MetricType,
						"new_value": containerinsightscommon.TypeNodeNVME,
					}},
				})
```

This is missing the Type label. Going to need to update the const file to add `NodeNVME` to the const file too

Reply:
Additionally, the `node_` prefix gets added by a helper function like: https://github.com/aws/amazon-cloudwatch-agent/blob/cb1da2f1e2d5b984e2ae27e927fcd262a83b454a/translator/translate/otel/processor/metricstransformprocessor/translator.go#L150 usually

Reply:
With that, you'll need to update https://github.com/aws/amazon-cloudwatch-agent/blob/main/internal/containerinsightscommon/util.go